tests/system/src/test/java/test/robot/test3d/PointLightIlluminationTest.java

```diff
@@ -66,9 +67,17 @@ public class PointLightIlluminationTest extends VisualTestBase {
     private static final double COLOR_TOLERANCE    = 0.07;
     private static volatile Scene testScene = null;
 
+    // Used to skip failing tests until JDK-8318985 is fixed
```

Comment:
we probably should add a note in JDK-8318985 description so we won't forget to revert these changes.

Reply:
Good idea. I added a comment to that JBS bug.